packages/frontend/web/components/lib/ArticleRenderer.tsx

```diff
@@ -35,5 +35,5 @@ export const ArticleRenderer: React.FC<{
             }
         })
         .filter(_ => _ != null);
-    return <>{output}</>;
+    return <div className="article-body-03f883b8">{output}</div>; // classname that space finder is going to target for in-body ads in DCR
```

Comment:
Hu, can you make a plan and stick it in a card to make that as dynamic as it should be?

Reply:
Replying on chat